tools/__tasks__/compile/javascript/babel.js

```diff
@@ -7,7 +7,13 @@ module.exports = {
     task: () =>
         execa(
             'babel',
-            [`${src}/javascripts`, '--out-dir', `${transpiled}/javascripts`],
+            [
+                `${src}/javascripts`,
+                '--out-dir',
+                `${transpiled}/javascripts`,
+                '--ignore',
+                '**/*.spec.js',
```

Comment:
Does this work ok? I was adding the same thing in my branch yesterday and my `.spec.js` files still ended up in there. Just tested it on this branch and get the same thing:

- Added `test.spec.js` to javascripts > projects > common > modules > commercial
- Ran `make test` and my jest test passes happily enough but then in the legacy test run I got:

```js
SyntaxError: Unexpected token ')'
  at static/transpiled/javascripts/projects/common/modules/test.spec.js:3
```

Is this babel still whacking it over or is there a copy task somewhere?